tokenserver/tests/test_memorynode.ini

```diff
@@ -13,6 +13,9 @@ node = https://example.com
 token_duration = 3600
 node_type_patterns =
   example:*example*
+spanner_entry  = https://spanner.example.com
+spanner_node_id = 800
```

Comment:
Something about naming these config entries after an implementation detail of the new backend seems unwise to me, but that's probably just my own unhealthy obsession with finding The Best Name for things.

Reply:
Hey, I named the spanner selection function `lucky_user` so, yeah, I'm terrible at naming things. Very happy to accept better suggestions.